cli/bench/deno_http.js

```diff
@@ -0,0 +1,68 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+async function main(addr) {
+  const rid = await Deno.http.createServer(addr);
+  console.log("Server listening on", addr);
+
+  while (true) {
+    const connRid = await Deno.http.accept(rid);
+    handleConn(connRid);
+  }
+}
+
+async function handleConn(connRid) {
+  while (true) {
+    const [connectionClosed, ...req] = await Deno.http.nextRequest(connRid);
+
+    handle(req);
+
+    if (connectionClosed) {
+      break;
+    }
+  }
+}
+
+async function handle(req) {
+  const [requestBodyRid, responseSenderRid, method, headers, url] = req;
```

Comment:
avoid using iterator protocol

```suggestion
  const { 0: requestBodyRid, 1: responseSenderRid, 2: method, 3: headers, 4: url } = req;
```